js/main.js

```diff
@@ -106,8 +107,13 @@ function goBack(event) {
   $homePage.className = 'homepage view';
   $viewPage.className = 'viewpage hidden';
   $weatherPage.className = 'weatherpage hidden';
+  $weatherContainer.remove();
+  $weatherHeading.remove();
+  // $imageContainer.remove();
   $mainHeading.className = 'heading view';
   $weatherHeading.className = 'hidden';
+  $headerColor.className = 'header normal';
+  window.location.reload(true);
```

Comment:
You should _**never**_ use `window.location.reload`. This is reloading the entire page. What is the purpose of this line?